templates/telemetry/clients_last_seen_v1/view.sql

```diff
@@ -15,6 +15,7 @@ SELECT
     ARRAY(SELECT * FROM UNNEST(environment_settings_intl_requested_locales.list)) AS environment_settings_intl_requested_locales,
     ARRAY(SELECT * FROM UNNEST(environment_settings_intl_system_locales.list)) AS environment_settings_intl_system_locales,
```

Comment:
For parquet the nested format will be restored on export for backwards compatibility.